web_development_101/the_front_end/project_html_css.md

```diff
@@ -2450,6 +2450,8 @@ Submit a pull request with a link to your solution in this [file](https://github
 * [Siddhanth's Solution (Easy)](https://github.com/vinecreeper888/google-homepage) - [View in Browser](https://vinecreeper888.github.io/google-homepage/)
 * [Christopher's Solution (Easy)](https://github.com/gandolfthewise/google-homepage) - [View in Browser](https://gandolfthewise.github.io/google-homepage/) 
 * [qgeo's Solution (Easy)](https://github.com/qgeo/google-homepage) - [View in Browser](https://qgeo.github.io/google-homepage/)
+* [elsmog's Solution (Easy)] (https://github.com/elsmog/google-homepage) - [View in Browser] (https://github.com/elsmog/google-homepage/)
+
```

Comment:
* [Nahabwe's Solution (Easy)] (https://https://musicmeetscode.github.io/google-homepage/) - [View in Browser] (https://github.com/musicmeetscode/google-homepage/)